test/TEST-35-LOGIN/test.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+# SPDX-License-Identifier: LGPL-2.1-or-later
+set -e
+
+TEST_DESCRIPTION="LOGIN"
+IMAGE_NAME="default"
+TEST_SAVE_JOURNAL="yes"
```

Comment:
If you want to get the journals from CentOS CI, you will have to make the test fail (e.g. by `exit 1` at the end), as CentOS CI doesn't store journals of passing tests, to conserve space.

Reply:
Also, maybe we should run all tests with `systemd.default_standard_output=journal+console`, so we always have the test output at hand no matter what, to avoid jumping through such hoops.

Reply:
> systemd.default_standard_output=journal+console

This enables all services logs on console, and it sounds quite noisy for me. Maybe, it is better to set it all test service with StandardOutput= and StandardError=.